projects/kubernetes/release/docker/kube-proxy-base/Dockerfile

```diff
@@ -26,8 +26,6 @@ COPY ATTRIBUTION.txt /ATTRIBUTION.txt
 ENTRYPOINT ["/go-runner"]
 
 
-FROM base as standard
-
 
 
 # Installing dependencies listed in upstream here: https://github.com/kubernetes/release/blob/master/images/build/debian-iptables/buster/Dockerfile
```

Comment:
It won't let me highlight all the lines in the PR, but do I need to delete everything from here to just before `FROM base as minimal`?